source/javascripts/homepage/_blog_posts.es6

```diff
@@ -0,0 +1,36 @@
+(function(window, $) {
+
+  function loadBlogPosts() {
+    const xhttp = new XMLHttpRequest();
+    xhttp.onreadystatechange = function() {
+      if (xhttp.readyState == 4 && xhttp.status == 200)
+        updatePosts(JSON.parse(xhttp.responseText));
+    };
+    xhttp.open("GET", "http://subvisual.co/blog/posts.json", true);
```

Comment:
you migth as well change this to `https` to avoid having a redirect all the time